### ISSUE TITLE
Improve UX during DocumentServer startup

### ISSUE THREAD

**Issue by dreua** [ONLYOFFICE/onlyoffice-humhub#35]
After a server reboot, the DocumentServer needs significant time to warm up. During this time, you can click "Edit" on documents in HumHub but all you get to see is the overly window with a big white rectangle as its content. It would be nice to have some information about what is going on and maybe a spinner or something while we wait. (With periodic refresh or refresh once the server is ready.)
